src/main/java/com/gluonhq/substrate/target/MacOSTargetConfiguration.java

```diff
@@ -92,6 +98,11 @@ String getAdditionalSourceFileLocation() {
 
             String staticLibPath = "-Wl,-force_load," + projectConfiguration.getJavafxStaticLibsPath() + "/";
             linkFlags.addAll(asListOfStaticLibraryLinkFlags(staticLibPath, javafxLibs));
+
+            if (projectConfiguration.hasWeb()) {
+                linkFlags.addAll(asListOfStaticLibraryLinkFlags(staticLibPath, macfxWebkit));
+                linkFlags.addAll(macfxWeblibs);
```

Comment:
Move the addition of the dynamic libraries outside of the `if (useJavaFX)` block to right above it:

    if (useJavaFX && projectConfiguration.hasWeb()) {
        linkFlags.addAll(macfxWeblibs);
    }

That way, the dynamic and static libraries are kept grouped together instead of spreading them out again over the entire link command.

Reply:
ok, but these are not dynamic libraries